### PR TITLE
Turkish language fixes

### PR DESCRIPTION
**Pull request by Zaryob**
Tamamlanmamış çeviriler tamamlandı.(Turkish)

translation tr.ts: Translations not having been completed have been completed. :smile:

### CODE REVIEW

File: recovery/translation_tr.ts [hunk 1]
```diff
@@ -564,7 +565,7 @@ Would you like to make this setting permanent?</source>
     <message>
         <location filename="multiimagewritethread.cpp" line="116"/>
         <source>Operating system cannot claim both primary partitions 2 and 4</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (lurch) [inline recovery/translation_tr.ts]: ?

**Comment by lurch** [comment]
@XECDesign is the current NOOBS maintainer :wink:  I'm sure this will get merged in due course...

**Comment by XECDesign** [comment]
Thanks, @Zaryob

**Comment by Zaryob** [comment]
> Thanks, @Zaryob

thank you for giving the opportunity to edit my language (Turkish) in my favorite project (Raspberry Pi) :smile:

**Comment by XECDesign** [comment]
If you're interested in getting more involved with the translation community, I think they would appreciate it:

https://www.raspberrypi.org/translate/

File: recovery/translation_tr.ts [hunk 2]
```diff
@@ -154,42 +154,42 @@ UYARI: Bu işlem SD kartınızdaki tüm verileri silecektir</translation>
...
         <location filename="initdrivethread.cpp" line="342"/>
         <source>Saving boot files to memory</source>
-        <translation type="unfinished"></translation>
+        <translation>Önyükleme dosyaları belleğe kaydediliyor</translation>
     </message>
     <message>
         <location filename="initdrivethread.cpp" line="345"/>
         <source>Error saving boot files to memory. SD card may be damaged.</source>
-        <translation type="unfinished"></translation>
+        <translation>Önyükleme dosyalarını belleğe kaydetmede hata yaşandı. SD kart zarar görmüş olabilir.<</translation>
```
Comment (lurch) [inline recovery/translation_tr.ts]: @Zaryob Somehow there was a duplicate less-than symbol (`<`) near the end of this line, which was causing problems.
I've fixed this in https://github.com/raspberrypi/noobs/commit/2f6a07e92b81e99f39cd3e274f2af663abca32aa

**Pull request merged**